### PR TITLE
Update UI to use Material Design

### PR DESCRIPTION
**Pull request by katelynsills**
Updates the UI to use Material Design. Also takes advantage of Material Design Component support for "snackbars" (messages that appear and disappear at the bottom of the page) and dialogs to direct and orientate the user.

Starting dialog:

Screen Shot 2020-10-15 at 6 52 46 PM

After clicking the button, the snackbar appears:

Screen Shot 2020-10-15 at 6 53 31 PM

After a successful offer result has been received, the following snackbar appears:

Screen Shot 2020-10-15 at 6 53 48 PM

### CODE REVIEW

File: ui/public/src/main.js [hunk 1]
```diff
@@ -24,13 +41,41 @@ export default async function main() {
...
+      case 'walletSuggestInstanceResponse': {
+        // TODO: handle appropriately
+        break;
+      }
+      case 'walletOfferAdded': {
+        // TODO: handle appropriately
+        break;
+      }
+      case 'walletOfferHandled': {
+        // TODO: handle appropriately
```
Comment (michaelfig) [inline ui/public/src/main.js]: This is where the "approve the offer" toast can go away.

File: ui/public/src/main.js [hunk 2]
```diff
@@ -24,13 +41,41 @@ export default async function main() {
...
+      case 'walletSuggestInstallationResponse': {
+        // TODO: handle appropriately
+        break;
+      }
+      case 'walletSuggestInstanceResponse': {
+        // TODO: handle appropriately
+        break;
+      }
+      case 'walletOfferAdded': {
+        // TODO: handle appropriately
```
Comment (michaelfig) [inline ui/public/src/main.js]: This is where the "approve the offer" toast can appear.

**Review by michaelfig — APPROVED** [review]
LGTM!  I'll set the base to my PR, and then the tests should pass.

File: ui/public/src/main.js (hunk 2 above)
Comment (katelynsills) [reply ui/public/src/main.js]: Done!

File: ui/public/src/main.js (hunk 1 above)
Comment (katelynsills) [reply ui/public/src/main.js]: Done!

**Pull request merged**